rasa_nlu_examples/extractors/flashtext_entity_extractor.py

```diff
@@ -0,0 +1,137 @@
+import os
+from typing import List, Type, Optional, Dict, Text, Any
+
+import rasa.shared.utils.io
+
+from rasa.nlu.components import Component
+from rasa.nlu.config import RasaNLUModelConfig
+from rasa.nlu.extractors.extractor import EntityExtractor
+from rasa.nlu.extractors.regex_entity_extractor import RegexEntityExtractor
```

Comment:
Are we using this internally?

Reply:
Which part do you mean? The first two are used for typing, the third is extended by `FlashTextEE`, and the last I don't use, so should remove :)

Reply:
```
from rasa.nlu.extractors.regex_entity_extractor import RegexEntityExtractor
```

The `RegexEntityExtractor` isn't used inside of this `FlashTextEntityExtractor`, or am I mistaken?

Reply:
Yes, please :)